docs/notes.md

```diff
@@ -603,4 +603,254 @@ sends a pair \\((L\_j,R\_j)\\) of points at each step \\(j = k\dots1\\). An
 additional and final step involves sending a pair of scalars
 \\((a^{(0)}\_0,b^{(0)}\_0)\\) and checking the final relation directly.
 
-[bulletproofs_paper]: https://eprint.iacr.org/2017/1066.pdf
+Aggregated Range Proof
+======================
+
+We want to create an aggregated range proof for `m` values that is more efficient to create and verify than `m` individual range proofs.
```

Comment:
Maybe this could lead by saying that the goal is to take advantage of the logarithmic size of the inner-product protocol?

As is I think it isn't quite right, because creating the aggregated range proof isn't any more efficient than individual proofs, and I'm not sure how verifying an aggregated proof compares to batch-verifying individual proofs. But the size is a win from aggregation.